### PR TITLE
chore: update story 11 to match sprint content

### PR DESCRIPTION
**Pull request by ekeren**
*By submitting this pull request, I confirm that my contribution is made under the terms of the 
[Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: docs/05-rfcs/stories/11-story.md [hunk 1]
```diff
@@ -61,10 +61,6 @@ resource TaskList {
    */
   inflight remove_tasks(id: str) {
     let content = this._bucket.get(id)
```
Comment (staycoolcall911) [inline docs/05-rfcs/stories/11-story.md:63]: warning: unused variable: `content`

File: docs/05-rfcs/stories/11-story.md [hunk 2]
```diff
@@ -74,7 +70,7 @@ resource TaskList {
     * @returns set of task id
     */
   inflight list_task_ids(): Set<str> {
-    return this._bucket.list();
+    return new Set<str>(this._bucket.list()); // pending #116
```
Comment (staycoolcall911) [inline docs/05-rfcs/stories/11-story.md:73]:
```suggestion
    return Set<str>(this._bucket.list()); // pending #1116
```

**Review by staycoolcall911 — APPROVED** [review]

**Comment by MarkMcCulloh** [comment]
note: due to https://github.com/winglang/wing/pull/1139, this file has been renamed from `11-story.md` -> `story-11.md`

File: docs/05-rfcs/stories/11-story.md (hunk 1 above)
Comment (ekeren) [inline docs/05-rfcs/stories/11-story.md]:
```suggestion
```

**Pull request merged**